gatsby-config.js

```diff
@@ -1,41 +1,63 @@
+const apiUrl = "https://api.adoptopenjdk.net/v3";
+
+// Available releases are hard-coded here to allow for the assets endpoint
+const availableVersions = [8, 9, 10, 11, 12, 13];
```

Comment:
You should be able to get this from https://api.adoptopenjdk.net/v3/info/available_releases

Reply:
You can, but you can't make async requests in the config file. Still figuring out the best way to get this...